### DISCUSSION TITLE
LTPBR symbology

### DISCUSSION THREAD

**Discussion by CHafen** [Riverscapes/RiverscapesXML#189] [Ideas]
@nick4rivers @joewheaton Here's an example of some structure symbology I came up with. The points are just points I added to QGIS to play with the symbology, so ignore that they aren't in the exact spots they should be compared to the structure lines. Let me know what you think.
![structures1](https://user-images.githubusercontent.com/39837888/127921728-cb3abe48-a191-47a9-a86b-0f55cbdebfb8.png)

**Comment by joewheaton** [comment]
So, cyan and magenta for the structures are good for viewing on a screen, but I think will not be all that cartographically pleasing for whatever we end up printing.  Part of me wonders if we didn't go for something that was blacks and whites and greys (e.g. outlines and highlights). I'm not sure. Maybe we can and should review a bunch of existing Anabranch reports and dump in screen shots of ideas and see what we like or do not like. Another consideration is whether the basemap will be imagery (like here... where whites and blacks and greys with highlights sometimes pop better) or a DEM, or a hillshade or there will be polygons underneath with some color. 

@nick4rivers do you have any vision or ideas here. I realize I have not thought this entirely through just yet. Perhaps we just run with this or something similarly simple and get going (we can change stuff easily). Also, don't forget Business Logic allows us to expose lots of the same things over and over (e.g. a polygon with fill vs. same polygon with hollow outline, vs. same polygon fill transparent).

**Reply by CHafen** [reply]
Thanks for the feedback @joewheaton. I think it would be a great idea to dump in screen shots from existing reports to see what we we like and don't like.

**Comment by joewheaton** [comment]
@CHafen and @nick4rivers, this is now becoming a top priority. I am going to engage you both on this but over in QRiS.
